backend/workhub/controllers/user_controller.js

```diff
@@ -78,4 +106,47 @@ exports.login = function(req,res){
 
 	})
 
+}
+
+
+exports.profileInfo = function(req,res){
+	var user_id = req.params.userId;
+
+
+
+	var firstName;
+	var lastName;
+	var desc;
+	var rating;
+
+	profile.findOne({
+		where: {user_id : user_id},
+		include: [{
+			model: user,
+			as: 'user',
+			where: {id : user_id}
+		}]
+	}).then(info => {
+		if (info){
+			firstName = info.user.firstName;
+			lastName = info.user.lastName;
+			desc = info.description;
+			rating = info.rating;
+
+			console.log(desc);
```

Comment:
I think, we can remove these logs; wdyt?

Reply:
yeah, not really necessary at this point

Reply:
```suggestion

```